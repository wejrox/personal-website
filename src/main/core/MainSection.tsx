import * as React from "react";
import Row from "reactstrap/lib/Row";
import Col from "reactstrap/lib/Col";

class MainSection extends React.Component {

    public render() {
        return (
            <Row>
                <Col>
                    Main Section
                </Col>
            </Row>
        );
    }
}

export default MainSection;