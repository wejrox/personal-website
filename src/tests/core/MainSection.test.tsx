import * as Enzyme from "enzyme";
import * as React from "react";
import MainSection from "../../main/core/MainSection";

describe("MainSection Component", () => {
    const wrapper = Enzyme.shallow(<MainSection/>);

    it("renders correctly", () => {
        expect(wrapper).toMatchSnapshot();
    });
});