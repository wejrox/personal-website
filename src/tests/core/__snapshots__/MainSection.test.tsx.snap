// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainSection Component renders correctly 1`] = `ShallowWrapper {}`;
